### DISCUSSION TITLE
Dead link in CONTRIBUTING.md

### DISCUSSION THREAD

**Discussion by pjmattingly** [serverless/serverless#11205] [General]
I was going through the Contributing guide (./CONTRIBUTING.md) and I found that this link is dead:

> Check [docs/anatomy.md](./docs/anatomy.md)

I found an article that seems to be correct: https://www.serverless.com/blog/anatomy-of-a-serverless-app/

But also, there's a Getting Started guide that seems to be more robust: https://www.serverless.com/framework/docs/tutorial

This seems like a good first pull request; That is, changing the link to one of these. Which link would be best to add to `CONTRIBUTING.md`?

**Comment by pgrzesik** [comment]
Hey @pjmattingly - thanks for reporting. I think none of the above would be fitting - previously the Anatomy document had information about the internal anatomy of the Framework from the perspective of a maintainer, not from a user. I think the best would be to just remove the `Anatomy of a Framework` section. Would you be interested in submitting a PR for that?

**Comment by pjmattingly** [comment]
Will do. I need to get used to the process, so that seems like a good first
step.

**Reply by pgrzesik** [reply]
Great to hear that 🙌 It definitely sounds like a really good first PR to get yourself familiar with the whole process 💯

**Reply by pjmattingly** [reply]
Okay, I think I did it right. I would appreciate any feedback you might have on: https://github.com/serverless/serverless/discussions/11209.

**Reply by pgrzesik** [reply]
Awesome job, thank you @pjmattingly 🙇
